.travis.yml

```diff
@@ -5,9 +5,14 @@ sudo: false
 cache:
     directory:
         - $HOME/.composer/cache/files
+        - $HOME/symfony-bridge/.phpunit
+
+env:
+    global:
+        - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
```

Comment:
defining this env variable allows to make the bridge install the PHPUnit in a cacheable location instead of downloading it in `vendor/symfony/phpunit-bridge/.phpunit` for which Travis caching of the folder would interfer with the Composer installation.

Btw, for local development, I added this in my ZSH config:

```bash
# Symfony PHPUnit bridge
export SYMFONY_PHPUNIT_DIR=~/.cache/symfony-phpunit-bridge
export SYMFONY_PHPUNIT_REMOVE=symfony/yaml
```

This is a nice way to share the PHPUnit downloaded by the bridge among all projects I have locally